src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -197,7 +197,8 @@
 - take a single parameter of type 'TestContext'
 - return type should be 'void', 'Task' or 'ValueTask'
 - not be 'async void'
-- not be a special method (finalizer, operator...).</value>
+- not be a special method (finalizer, operator...)
+- the attribute should have an inheritance behavior parameter if the containing class is abstract.</value>
```

Comment:
It's more than simply value is set, it should be set to `BeforeEachDerivedClass`.

`- 'InheritanceBehavior.BeforeEachDerivedClass' attribute parameter should be specified if the class is 'abstract'.`